script.js

```diff
@@ -81,12 +83,51 @@ function checkWin() {
   return false;
 }
 
+function checkDraw() {
+  if (
+    gameSpaces[0].innerHTML !== "" &&
+    gameSpaces[1].innerHTML !== "" &&
+    gameSpaces[2].innerHTML !== "" &&
+    gameSpaces[3].innerHTML !== "" &&
+    gameSpaces[4].innerHTML !== "" &&
+    gameSpaces[5].innerHTML !== "" &&
+    gameSpaces[6].innerHTML !== "" &&
+    gameSpaces[7].innerHTML !== "" &&
+    gameSpaces[8].innerHTML !== ""
+  ) {
+    return true;
+  }
+  return false;
+}
+
+function disableGameSpace(trueFalse) {
```

Comment:
you may want to rename is `disableGameSpaces` since it doesn't just do one space